### PR TITLE
Put ci.centos job links on build description

### PR DESCRIPTION
**Pull request by ehelms**
Also add simple test jobs for ci.theforeman.org and ci.centos

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-ci-centos-simple-test.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-ci-centos-simple-test.yml]: Do we want to keep this or was it just for debugging?

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy [hunk 2]
```diff
@@ -35,7 +35,12 @@ pipeline {
                 withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'jenkins-centos', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
                     runPlaybook(
                         playbook: 'ci/centos.org/ansible/jenkins_job.yml',
-                        extraVars: ["jenkins_job_name=foreman-nightly-test", "jenkins_username=${env.USERNAME}", "jenkins_password=${env.PASSWORD}"],
+                        extraVars: [
+                            "jenkins_job_name=foreman-nightly-test",
+                            "jenkins_username=${env.USERNAME}",
+                            "jenkins_password=${env.PASSWORD}"
+                            "jenkins_job_link_file=${env.WORKSPACE}jobs/foreman-nightly-test"
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy]: Does `env.WORKSPACE` include a trailing slash? Note this is repeated a few times in the PR

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy]: Good catch, I missed that when updating from test job.

File: puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-ci-centos-simple-test.yml (hunk 1 above)
Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-ci-centos-simple-test.yml]: I would like to keep it. This gave an easy (and quick) way to test the interaction without having to run any one of our production level jobs.

**Comment by ehelms** [comment]
Updated

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/ci_centos_integration_test.groovy [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    stages {
+        stage('Run simple ci.centos.org test') {
+            steps {
+                deleteDir()
+                git_clone_foreman_infra()
+                sh "mkdir jobs"
+
+                withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/ci_centos_integration_test.groovy]: Almost feels like we could introduce a runCentosCIPlaybook helper

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/ci_centos_integration_test.groovy]: Yep -- I just didn't want to re-factor here given this was complex enough change.

**Pull request merged**